src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -152,7 +153,13 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 		if (this.connection) {
 			let baseName = this.connection.serverName + ':' + this.connection.databaseName;
 			let advancedOptions = this._connectionService.getNonDefaultOptions(this.connection);
-			fullTitle = fullTitle + ': ' + baseName + advancedOptions;
+			fullTitle = fullTitle + ': ' + baseName + advancedOptions + '\n';
+			if (this.sessionName) {
+				fullTitle += nls.localize('profilerInput.sessionNameLoaded', 'Session Name: {0}', this.sessionName);
+			}
+			else {
+				fullTitle += nls.localize('profilerInput.sessionNameNotLoaded', 'Session Name: not available');
```

Comment:
My suggestion would be to not add this. Just have the session name if present, else ignore.

Reply:
You're right, we would almost never see this.